README.md

```diff
@@ -0,0 +1,145 @@
+---
+permalink: /guides/nexus-repository-integration-with-appsody/
+layout: guide-markdown
+title: Nexus Repository Integration
+duration: 40 minutes
+releasedate: 2020-02-21
+description: Learn how to use Nexus Repository in Codewind
+tags: ['Codewind', 'Java', 'Nodejs']
+guide-category:
+---
+# Nexus Repository Integration
+//
```

Comment:
You need to use HTML escape characters to hide the license i.e. &lt;!-- and --&gt;

Reply:
fixed